### DISCUSSION TITLE
Welcome to pgMail Discussions!

### DISCUSSION THREAD

**Discussion by captbrando** [captbrando/pgMail#8] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Nandhini-Madan** [comment]
Can you add screenshots of the email and sample email you get.

**Reply by captbrando** [reply]
Hey there @Nandhini-Madan! Apologies I missed this earlier. I could perhaps get some screenshots in place when I have a moment. They will look however you want them to look based on how the system works.
